api/permissions/permission_service.py

```diff
@@ -220,6 +225,36 @@ def get_base_permission_filter(
     return user_filter | group_filter | role_filter
 
 
+def get_object_id_from_base_permission_filter(
+    user: "FFAdminUser",
+    for_model: Union[Organisation, Project, Environment] = None,
+    permission_key: str = None,
+    allow_admin: bool = True,
+    tag_ids=None,
+) -> Set[int]:
+    object_ids = set()
+    user_filter = get_user_permission_filter(user, permission_key, allow_admin)
+    object_ids.update(
+        list(for_model.objects.filter(user_filter).values_list("id", flat=True))
+    )
+
+    group_filter = get_group_permission_filter(user, permission_key, allow_admin)
+
+    object_ids.update(
+        list(for_model.objects.filter(group_filter).values_list("id", flat=True))
+    )
+    if settings.IS_RBAC_INSTALLED:
```

Comment:
```suggestion
    if settings.IS_RBAC_INSTALLED:  # pragma: no cover
```

This will exclude all the code from the if branch.